### PR TITLE
AA-0003 fixed the Call to undefined method PhpParser\Node\Expr\Variab…

### PR DESCRIPTION
**Pull request by ahmed-abid**
got this issue :
[Error] Call to undefined method PhpParser\Node\Expr\Variable::toString()

while running some tests using AspectMock caused by this line:

https://github.com/goaop/framework/blob/72521d026692131f4052c1d7b3dad15c846cdd0f/src/Instrument/Transformer/MagicConstantTransformer.php#L94

a suggested solution here: https://github.com/nikic/PHP-Parser/pull/596/files

we can do something here as well as following

### CODE REVIEW

File: src/Instrument/Transformer/MagicConstantTransformer.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ private function wrapReflectionGetFileName(StreamMetaData $metadata): void
         /** @var MethodCall[] $methodCalls */
         $methodCalls = $methodCallFinder->getFoundNodes();
         foreach ($methodCalls as $methodCallNode) {
-            if ($methodCallNode->name->toString() !== 'getFileName') {
+            if ($methodCallNode->name instanceof \PhpParser\Node\Identifier && $methodCallNode->name->toString() !== 'getFileName') {
```
Comment (lisachenko) [inline src/Instrument/Transformer/MagicConstantTransformer.php]: Please use import of class name

**Review by lisachenko — CHANGES_REQUESTED** [review]
Hi, thank you fo sending this PR, please fix small notice in it.

File: src/Instrument/Transformer/MagicConstantTransformer.php (hunk 1 above)
Comment (ahmed-abid) [reply src/Instrument/Transformer/MagicConstantTransformer.php]: Please review

**Review by lisachenko — APPROVED** [review]

**Comment by lisachenko** [comment]
Squashed and merged into master, thank you!

**Comment by ddinchev** [comment]
@lisachenko I'm also hitting the same issue. Can you release this fix in 2.3.x?

**Comment by lisachenko** [comment]
@ddinchev This fix was merged into the master branch only, so you should not expect it to be in 2.x branch. Please check **master** branch instead, aka **3.x-dev**

**Pull request merged**